exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
I'm tracking `frameIndex` and `framePage` on the `participate.survey.index` so that I know what frame/page the user is on when they hit the back button